recurring_contract/data/contract_expire_cron.xml

```diff
@@ -1,22 +1,20 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-    Copyright (C) 2014 Compassion (http://www.compassion.ch)
-    @author Cyril Sester <cyril.sester@outlook.com>
-    The licence is in the file __manifest__.py
--->
-<odoo>
-    <data noupdate="1"> <!-- noupdate = 1 for the 'active' field -->
-        <record id="recurring_contract_expire_cron" model="ir.cron">
-            <field name="name">Check end of contract</field>
-            <field name="active" eval="True"/>
-            <field name="user_id" ref="base.user_root"/>
-            <field name="interval_number">1</field>
-            <field name="interval_type">days</field>
-            <field name="numbercall">-1</field> <!-- don't limit the number of calls -->
-            <field name="doall" eval="False"/>
-            <field name="model" eval="'recurring.contract'"/>
-            <field name="function" eval="'end_date_reached'" />
-            <field name="args" eval="'()'"/>
-        </record>
-    </data>
-</odoo>
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2014 Compassion (http://www.compassion.ch)
+    @author Cyril Sester <cyril.sester@outlook.com>
+    The licence is in the file __manifest__.py
+-->
+<odoo>
+    <data noupdate="1">
+        <record id="recurring_contract_expire_cron" model="ir.cron">
+            <field name="name">Check end of contract</field>
+            <field name="active" eval="True"/>
+            <field name="user_id" ref="base.user_root"/>
+            <field name="interval_number">1</field>
+            <field name="interval_type">days</field>
+            <field name="numbercall">-1</field>
+            <field name="model_id" ref="model_recurring_contract"/>
+            <field name="function" eval="'end_date_reached'" />
```

Comment:
Why use double quote and quote together?

Reply:
Because of eval which executes python code and here we want a string. Best would be to remove eval and put the string as a normal xml tag.